Sources/MapGL/Model/GeographicalBounds.swift

```diff
@@ -16,6 +16,31 @@ public struct GeographicalBounds: Decodable {
 		self.northEast = northEast
 		self.southWest = southWest
 	}
+
+	///Extend the bounds to include a given point.
+	public func extend(point: CLLocationCoordinate2D) -> GeographicalBounds {
+		return self.extend(points: [point])
+	}
+
+	///Extend the bounds to include given points.
+	public func extend(points: [CLLocationCoordinate2D]) -> GeographicalBounds {
+		var minLon = southWest.longitude
+		var minLat = southWest.latitude
+		var maxLon = northEast.longitude
+		var maxLat = northEast.latitude
+
+		points.forEach {value in
```

Comment:
Я честно не считаю, что использование `.forEach` имеет плюсы над `for ... in` в неленивом случае.
```suggestion
		for value in points {
```
Стоит также переименовать `value` в `point`.